src/pages/viewer-authentication-api-getting-started.mdx

```diff
@@ -5,12 +5,7 @@ description: Deprecated - Getting started
 
 <InlineNotification kind="warning">
 
-**Deprecation Warning:** dolor sit amet,
-[consectetur adipiscing](/components/InlineNotification) elit. Curabitur ac odio
-arcu. Vestibulum egestas eleifend porttitor. Quisque malesuada pulvinar
-pellentesque. Nunc dictum odio eu enim venenatis fringilla. Nunc finibus enim
-dui, a tempus quam commodo vitae. Donec non eros gravida dolor porta suscipit
-non vel quam.
+**Deprecation Warning:** This API supported until 03/01/2022. Please use new version of the [Viewer Authentication](/viewer-authentication-api-v2-getting-started).
```

Comment:
Suggestion:

>This API supported until 03/01/2022. Please use new version of the [Viewer Authentication](/viewer-authentication-api-v2-getting-started).

This API is deprecated and it is scheduled to be removed on 03/01/2022. Please use (or migrate to) the new version of the [Viewer Authentication](/viewer-authentication-api-v2-getting-started).